deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -924,3 +924,24 @@ def modify_file(path, name, new_content=None, is_binary=False):
     modify_file_group(path, name)
     modify_file_permission(path, name)
     modify_file_win_attributes(path, name)
+
+
+def create_temp_file(content: str) -> str:
+    """
+    Create a temporary file with the specified content.
+
+    Args:
+        content (str): The content to be written to the temporary file.
+
+    Returns:
+        str: The path to the created temporary file.
+    """
+    try:
+        fd, temp_file_path = tempfile.mkstemp(text=True)
+        with os.fdopen(fd, 'w', newline='\n') as temp_file:
+            temp_file.write(content)
+        return temp_file_path
+    except Exception as e:
+        print(f"Error creating temporary file: {e}")
+        raise
```

Comment:
If it fails to create the file, the exception should already be raised, if you want to change the exception log:

```suggestion
    except Exception as e:
        raise e(f"Error creating temporary file: {temp_file_path}")
```